src/db/init_database.js

```diff
@@ -71,6 +71,11 @@ export async function populateTools() {
     documentationUrl: 'http://docs.sequelizejs.com/',
     logoSmallUrl: 'http://docs.sequelizejs.com/manual/asset/logo-small.png'
   });
+  await sequelize.Tool.create({
+    name: 'Skadi',
+    toolType: sequelize.ToolType.SKADI,
```

Comment:
We could make an actual type like monitoring.

Reply:
Yes, we could. However, Skadi is kind of on its own and a tool provided by, which the user has no option for. Going to leave it. Maybe we'll change it one day...